src/mfa/webauthn/controller/attestation.ts

```diff
@@ -0,0 +1,95 @@
+import Controller from '@curveball/controller';
```

Comment:
This is just copying the content of `src/mfa/webauthn/controller/register.ts` into this new file.